addons/stock_dropshipping/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,5 @@
+---Fields in module 'stock_dropshipping'---
+stock_dropshipping / purchase.order.line      / sale_line_id (many2one)       : NEW relation: sale.order.line
```

Comment:
You have to fill this field with the proper value. Take it from the associated procurement.order